runtime/runtime/src/state_viewer/mod.rs

```diff
@@ -264,12 +264,13 @@ impl TrieViewer {
             Err(errors::CallFunctionError::VMError { error_message: message })
         } else {
             let outcome = outcome.unwrap();
-            debug!(target: "runtime", "(exec time {}) result of execution: {:#?}", time_str, outcome);
-            logs.extend(outcome.logs);
-            let mut result = vec![];
-            if let ReturnData::Value(buf) = &outcome.return_data {
-                result = buf.clone();
-            }
+            debug!(target: "runtime", "(exec time {}) result of execution: {}", time_str, outcome);
+            let VMOutcome { logs: outcome_logs, return_data, .. } = outcome;
+            logs.extend(outcome_logs);
+            let result = match return_data {
+                ReturnData::Value(buf) => buf,
+                _ => vec![],
```

Comment:
Nit, but I'd write this explicitly as 

```suggestion
                ReturnData::ReceiptIndex(_) | ReturnData::None => vec![],
```

as per this classical lovely interlaced advice: https://youtu.be/-J8YyfrSwTk?t=1819